.gitattributes

```diff
@@ -1,13 +0,0 @@
-* text=auto eol=lf
-
-extlibs/**/* -text -eol linguist-vendored
-cmake/toolchains/android.toolchain.cmake -text -eol
-
-*.png -text -eol
-*.jpg -text -eol
-*.wav -text -eol
-*.ogg -text -eol
-*.flac -text -eol
-*.tff -text -eol
-*.icns -text -eol
-*.rtf -text -eol
```

Comment:
Please don't remove this file.

Git can be configured in different ways to deal with line endings. The `.gitattributes` files ensures that line endings are handled correctly for the repository (e.g. not enforcing different line endings in the extlibs directory).

See also https://git-scm.com/docs/gitattributes

Reply:
That was an accident sorry! I deleted that file to see what would happen then probably got distracted and forgot to undo that change so it got accidentally committed.